inventory/allinone-ha.yaml

```diff
@@ -20,7 +20,7 @@ all:
             nodepool-builder-slave:
               hosts: allinone02
               vars:
-                service_state: stopped
+                nodepool_service_nodepool_builder_state: stopped
```

Comment:
Maybe named this in short like 'nodepool_builder_state'? and consider to rename for all of following vars.

Reply:
this var name is defined in the role `https://git.openstack.org/openstack/ansible-role-nodepool`, here we just override this variable value to `stopped`

Reply:
ok, thanks.